tests/trinity/core/integration_test_helpers.py

```diff
@@ -96,3 +109,57 @@ class FakeAsyncChain(MiningChain):
     coro_import_block = coro_import_block
     coro_validate_chain = async_passthrough('validate_chain')
     coro_validate_receipt = async_passthrough('validate_receipt')
+    chaindb_class = FakeAsyncChainDB
+
+
+class ByzantiumTestChain(FakeAsyncChain):
+    vm_configuration = ((0, ByzantiumVM),)
+    network_id = 999
+
+
+FUNDED_ACCT = keys.PrivateKey(
+    decode_hex("49a7b37aa6f6645917e7b807e9d1c00d4fa71f18343b0d4122a4d2df64dd6fee"))
+
+
+def load_mining_chain(db):
+    class POWByzantiumVM(POWMiningMixin, ByzantiumVM):
+        pass
+
+    GENESIS_PARAMS = {
+        'coinbase': constants.ZERO_ADDRESS,
+        'difficulty': 5,
+        'gas_limit': 3141592,
+        'timestamp': 1514764800,
+    }
+
+    GENESIS_STATE = {
+        FUNDED_ACCT.public_key.to_canonical_address(): {
+            "balance": 100000000000000000,
+        }
+    }
+
+    return build(
+        FakeAsyncChain,
+        fork_at(POWByzantiumVM, 0),
```

Comment:
You can do this as:

```python
return build(
  FakeAsyncChain,
  byzantium_at(0),
  enable_pow_mining(),
  ...,
)
```

Minor improvement but removes the need for the local `POWByzantiumVM` class.